### ISSUE TITLE
Steam games working fine in Desktop Mode but not launching in GamingMode (DX11 error)

### ISSUE THREAD

**Issue by sonalder-darlene** [ChimeraOS/chimeraos#875]
I had three games (bought on Steam) that refused to launch in GamingMode but when I go to Desktop Mode I can launch them without any errors. It isn't conveniant because I installed ChimeraOS to get a console experience so I tried to fixed them and only succesfully fixed one of them.

## The error :
A giant pop-up window with the following text :

### Couldn't switch to requested monitor resolution
Details:
Switching to resolution 3840x2160 failed, trying lower one
All resolution switches have failed
Screen: DX11 could not switch resolution (3840x2160 fs=0 hz=0)

## The games :

- Stick Fight: The Game (fixed)
- Heavenly Bodies
- Tools Up!

## What I have tried :

- Switching to different proton build (Experimental, GE, etc...)
- Launching in Desktop Mode (IT WORKS)
- Launch option : PROTON_USE_WINE3D=1 %command% (worked for Stick Fight)
- Launch option : -nobatch -no-graphics (The game launch, I can hear the menu music but my TV is only showing the steam loading screen with buttons at the bottom (STEAM Menu, B Force to quit))
- Launch option : -fullscreen=0 (and others but no success)

**Comment by sonalder-darlene** [comment]
I am using AMD GPU and CPU and using a high refresh rate (VRR) 4K TV over HDMI if this matters

Ryzen 5 3600
MSI Radeon RX 5700XT

**Comment by tolsac** [comment]
I have exactyle the same problem, on a 4K TV LG G3 but the game that doesnt launch is Overcooked 2

@sonalder-darlene did you manage to fix the problem ?

**Comment by alkazar** [comment]
This sounds like the games are unable to work with the native resolution of your display. You could try the following:
 - set a different resolution in the game's settings/properties inside the steam client
 - run the game in desktop mode and adjust the resolution settings from inside the game

**Comment by sonalder-darlene** [comment]
> I have exactyle the same problem, on a 4K TV LG G3 but the game that doesnt launch is Overcooked 2
> 
> @sonalder-darlene did you manage to fix the problem ?

Hi @tolsac unfortunately I did not manage to fix the issue on my device

**Comment by sonalder-darlene** [comment]
> This sounds like the games are unable to work with the native resolution of your display. You could try the following:
> 
>     * set a different resolution in the game's settings/properties inside the steam client
> 
>     * run the game in desktop mode and adjust the resolution settings from inside the game

Hi, thanks for your reply, I did try to change the game's resolution from steam game's properties without any success. I did not try to change the resolution from game settings as it just works on desktop mode but will definetly take a look into that !

**Comment by sonalder-darlene** [comment]
No luck with that either still the same error just showing a lower resolution in the message

**Comment by sonalder-darlene** [comment]
I'll add to that list of games with issue a new one that has a similar experience but with a different error message. It is Rayman Legends.
The UbisoftConnect launcher show up correctly, I can log in without issues but when I start the game it shows me this error pop up:

Create DirectX device Failed 10 times!
please report this error code to get support : -2005530518

When I click Next/OK (don't remember) I get this error pop up :

May your drivers are outdated
OK

I tried to install some DX9 stuff through ProtonTricks, change to windowed with several methods and different Proton versions and launch option without success.

The game launch no issue in Desktop Mode but keep showing this error in UbisoftConnect. That's why I didn't create a new issue as it's the same problem and seems related to DirectX (not 11 this time)

**Comment by sonalder-darlene** [comment]
As mentionned in this issue : https://github.com/ChimeraOS/chimeraos/issues/966
A workaround that is better than going in Desktop Mode is to disabled HDR in the display settings of steam. It did works for all of the games I had issue with except for Rayman Legends.

@tolsac

**Comment by sonalder-darlene** [comment]
> > As mentionned in this issue : [#966](https://github.com/ChimeraOS/chimeraos/issues/966) A workaround that is better than going in Desktop Mode is to disabled HDR in the display settings of steam. It did works for all of the games I had issue with except for Rayman Legends.
> > [@tolsac](https://github.com/tolsac)
> 
> Have you been able to run Rayman Legends since?

No, I have accepted that it won't work on my living room PC and finished it on the Steam Deck, didn't tried since.

However I had no issue with Overcooked 2. @tolsac
